libs/application-generic/src/usecases/workflow/update-workflow/update-workflow.usecase.ts

```diff
@@ -79,7 +79,9 @@ export class UpdateWorkflow {
         `Notification template with id ${command.id} not found`
       );
 
-    let updatePayload: Partial<NotificationTemplateEntity> = {};
+    let updatePayload: Partial<NotificationTemplateEntity> = {
+      preferences: command.preferences,
+    };
```

Comment:
It's possible that older `@novu/framework` versions are used during sync that don't have `command.preferences`, therefore we should follow the existing pattern of checking whether the property exists and add the property to the payload conditionally.